### PR TITLE
[next] First Iteration of Admin

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- First iteration over the Admin Client
- Implements `found` `found-relay`
- Routing to handle multiple views  -   MODERATE, COMMUNITY, STORIES, CONFIGURE 

## How do I test this PR?
- Go to http://localhost:8080/admin/moderate

### CODE REVIEW

File: config/watcher.ts [hunk 1]
```diff
@@ -22,6 +22,9 @@ const config: Config = {
         "core/client/auth/**/*.ts",
         "core/client/auth/**/*.tsx",
         "core/client/auth/**/*.graphql",
+        "core/client/admin/**/*.ts",
+        "core/client/admin/**/*.tsx",
+        "core/client/admin/**/*.graphql",
```
Comment (alinhle) [inline config/watcher.ts]: Not sure how the `auth` lines got here, but `npm run compile:relay-stream` does not touch these files. We can delete lines 22-27.

File: src/core/server/app/router.ts [hunk 2]
```diff
@@ -49,6 +50,9 @@ async function createTenantRouter(app: AppOptions, options: RouterOptions) {
   // Setup auth routes.
   router.use("/auth", createNewAuthRouter(app, options));
 
+  // Handle the admin handler.
+  router.get("/admin", adminHandler);
```
Comment (alinhle) [inline src/core/server/app/router.ts]: I let @wyattjoh the server part :-)

**Review by alinhle — CHANGES_REQUESTED** [review]
As briefly talked about. In the `admin` target we have full control over the location as we are not in an iframe nor in a popup and so instead of writing our own view router, we can rely on much more powerful libraries that helps us manage the various routes of the admin. One that comes with Relay modern integration would be https://github.com/relay-tools/found-relay

**Review by wyattjoh — APPROVED** [review]
Server side looks good to me, I cleaned up a few things!

File: src/types/farce.d.ts [hunk 3]
```diff
@@ -0,0 +1,4 @@
+declare module "farce" {
```
Comment (alinhle) [inline src/types/farce.d.ts:1]: Let's add a todo comment here `// TODO: provide better typings.`

File: src/core/client/admin/components/App.tsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import React, { StatelessComponent } from "react";
+import { HorizontalGutter } from "talk-ui/components";
+import styles from "./App.css";
+import Navigation from "./Navigation";
+
+const App: StatelessComponent = ({ children }) => (
+  <div className={styles.root}>
+    <HorizontalGutter>
```
Comment (alinhle) [inline src/core/client/admin/components/App.tsx]: `HorizontalGutter` can be used as root and we would save one unnecessary `div`, e.g. `<HorizontalGutter className={styles.root}>`

File: src/core/client/admin/components/Community.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React, { StatelessComponent } from "react";
+import { HorizontalGutter, Typography } from "talk-ui/components";
+
+const Community: StatelessComponent = ({ children }) => (
+  <HorizontalGutter>
```
Comment (cvle) [inline src/core/client/admin/components/Community.tsx]: This container seems obsolete, this is repeated in other files too.

**Review by alinhle — CHANGES_REQUESTED** [review]

**Review by alinhle — APPROVED** [review]

**Pull request merged**